### PR TITLE
Add endpoints to whatsapp broadcast

### PR DESCRIPTION
**Pull request by lucaslinhares**

### CODE REVIEW

File: temba/msgs/models.py [hunk 1]
```diff
@@ -140,7 +142,7 @@ def create(
             text = {base_language: text}
 
         assert groups or contacts or contact_ids or urns, "can't create broadcast without recipients"
-        assert base_language in text, "base_language doesn't exist in text translations"
+        # assert base_language in text, "base_language doesn't exist in text translations"
```
Comment (paulobernardoaf) [inline temba/msgs/models.py:145]: Shouldn't base_language be still asserted in default broadcasts?

File: temba/api/v2/views.py [hunk 2]
```diff
@@ -4063,6 +4238,12 @@ def get_serializer_context(self):
         context["is_zapier"] = "Zapier" in self.request.META.get("HTTP_USER_AGENT", "")
         return context
 
+    # def post(self, request, *args, **kwargs):
```
Comment (paulobernardoaf) [inline temba/api/v2/views.py:4241]: remove dead code

**Review by paulobernardoaf — CHANGES_REQUESTED** [review]

File: temba/msgs/models.py (hunk 1 above)
Comment (lucaslinhares) [reply temba/msgs/models.py]: You're right, fixed!

File: temba/api/v2/views.py (hunk 2 above)
Comment (lucaslinhares) [reply temba/api/v2/views.py]: Removed

**Review by paulobernardoaf — APPROVED** [review]

**Pull request merged**